depends/packages/qt.mk

```diff
@@ -45,6 +45,7 @@ $(package)_config_opts += -no-ico
 $(package)_config_opts += -no-iconv
 $(package)_config_opts += -no-kms
 $(package)_config_opts += -no-linuxfb
+$(package)_config_opts += -no-libjpeg
```

Comment:
Why can't we do changes in upstream build-aux/m4/bitcoin_qt.m4? imo let's drop this PR if it's partial

Reply:
The changes in `build-aux/m4/bitcoin_qt.m4` are in a part of the file we do not have